.github/workflows/ci.yml

```diff
@@ -11,13 +11,37 @@ on:
       - "**.md"
 
 jobs:
+  # In reality, this will not be completely accurate as only when a Docker image is deployed to GCP
+  # we will truly have a deployed release. Nevertheless, this will be useful once it is deployed
+  # since it will help add commits to releases
+  release:
+    if: ${{ github.ref == 'refs/heads/main' }}
+    name: Create a Sentry release
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # This is necessary for the a release to include commits
+          fetch-depth: 0
+
+      - name: Create Sentry release
+        uses: getsentry/action-release@master
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
```

Comment:
The auth token is generated thanks to this integration: https://sentry.io/settings/sentry/developer-settings/sentry-github-actions-app-release-integration-7bc999/